ui/App.svelte

```diff
@@ -22,7 +22,9 @@
   import Onboarding from "./Screen/Onboarding.svelte";
   import DesignSystemGuide from "./Screen/DesignSystemGuide.svelte";
   import Discovery from "./Screen/Discovery.svelte";
-  import Modal from "./Modal";
+  import ModalNewProject from "./Modal/NewProject.svelte";
+  import ModalSearch from "./Modal/Search.svelte";
+  import ModalShortcuts from "./Modal/Shortcuts.svelte";
```

Comment:
It would be separate such cosmetic changes into separates PRs. And I'd prefer to undo this.

Reply:
This is not a cosmetic change, it's a structural one. Given the overhead of getting a  change reviewed, approved and kept in sync with mainline, I made the call to include it here. As I touched the part of the codebase already. It does not distract or obfuscate the other changes and it should be fine to approve it along the rest.

Reply:
Sorry for the wrong categorization on my side. In any case, still not fond of this change. What's the context and motivation here?

Reply:
It's the removal of the unnecessary bookkeeping in `index.js` plus dropping the superfluous Modal suffix.

Reply:
Yeah, I don't dislike dropping the bookkepping but the namespacing I do appreciate. The prefix is staying around as you see, though less elegant. Happy to talk about this another time.